### PR TITLE
add idgrp1.tst and small-7-8-9.tst

### PR DESCRIPTION
**Pull request by wilfwilson**
I added some tests for layers 7, 8, 9 in the hopes of increasing code coverage. I'm about to get on a plane can't add more details, but I thought I'd make this PR now in case you would want to merge it before making a new release.

Since travis wasn't working last night I'm not sure what code coverage this gives yet.

### CODE REVIEW

File: tst/idgrp1.tst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+1
+
+# 3
+gap> ID_GROUP_FUNCS[3](Group([(1,2), (1,2,3)]), SMALL_AVAILABLE(6));
+1
+gap> ID_GROUP_FUNCS[3](Group([(1,2)(3,4,5)]), SMALL_AVAILABLE(6));
+2
+
+# 4
+gap> ID_GROUP_FUNCS[4](Group([(1,2,3,4,5,6,7,8)]), SMALL_AVAILABLE(8));
```
Comment (markuspf) [inline tst/idgrp1.tst]: There's either a line missing here, or a second semicolon

File: tst/idgrp1.tst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+gap> ID_GROUP_FUNCS[4](Group([(1,2,3,4), (5,6,7,8)]), SMALL_AVAILABLE(8));
+
+#
+gap> G := SmallGroup(100949, 5);;
+gap> H := PcGroupCode(CodePcGroup(G), 100949);;
+gap> IdSmallGroup(H);
+[ 100940, 5 ]
+
```
Comment (markuspf) [inline tst/idgrp1.tst]: This is a typo, both versions (package, and component correctly) return `[ 100949, 5]`.

**Comment by wilfwilson** [comment]
Thanks @markuspf - those kinds of things are what my brain does when it's tired!

**Comment by markuspf** [comment]
No worries, I am too happy about you providing these tests to be complaining about small mistakes like that.

**Comment by wilfwilson** [comment]
I'll adjust the tests another time so that they work in 32bit

**Comment by wilfwilson** [comment]
@markuspf I think it's sorted.

**Pull request merged**